README.md

```diff
@@ -139,10 +142,70 @@ mysql --user=ci_user --password=ci_pw
 Use `^C` to stop the running MySQL container,
 or -- if you used the detached flag `-d` with `docker-compose up` -- use `docker-compose down`.
 
-Data in the MySQL database will persist after the container is stopped. 
-The MySQL data is stored in a volume mapped to the `.data/` directory in the project. 
+Data in the MySQL database will persist after the container is stopped.
+The MySQL data is stored in a volume mapped to the `.data/` directory in the project.
 To completely reset the database, delete the `.data` directory.
 
+
+
+##### A Typical Development Cycle With Docker
+
+1. Build images for all services…
+
+    ```
+    docker-compose build
+    ```  
```

Comment:
For consistency with @jonespm's PR (#73), we should probably use language flags (e.g. `bash`).

Reply:
I'll use `sh` to be shell agnostic.

Reply:
I commented on another issue (In MWrite I think) for the purposes of language sh/zsh/shell-script/bash [are all aliases for the same thing.](https://github.com/github/linguist/blob/40992ba7f86889f80dfed3ba95e11e1082200bad/lib/linguist/languages.yml#L4984) It doesn't matter which one you use in Markdown here.

Reply:
Cool.  Then `sh` is a good choice.  Thanks!